### PR TITLE
How to Secure Nginx with Let's Encrypt on CentOS 7

### PR DESCRIPTION
**Pull request by 1817666**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: MohammadHossein Fakhraei eng.fakhraei@gmail.com

### CODE REVIEW

File: tutorials/secure-nginx-with-let-s-encrypt-on-centos-7/01.en.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [MohammadHossein Fakhraei eng.fakhraei@gmail.com]
```
Comment (n-se) [inline tutorials/secure-nginx-with-let-s-encrypt-on-centos-7/01.en.md]: I know you always did it like this, but the correct sign off would look like `Firstname Lastname <mail@whatever.com>`. Maybe you wanna use the correct form in the future? ;)

File: tutorials/secure-nginx-with-let-s-encrypt-on-centos-7/01.en.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+```
+
+The certbot Let's Encrypt client is now installed and ready to use.
+
+## Step 4 - Obtaining a Certificate
+
+Certbot provides a variety of ways to obtain SSL certificates, through various plugins. The Nginx plugin will take care of reconfiguring Nginx and reloading the config whenever necessary:
+
+```bash
+certbot --nginx -d example.com -d www.example.com
```
[the remark this answers is not in the document]
Comment (1817666) [reply tutorials/secure-nginx-with-let-s-encrypt-on-centos-7/01.en.md]: Yes, its OK.
Rsa key size 4096 is better than 2048 :)
Thanks.

Edited.

[the remark this answers is not in the document]
Comment (1817666) [reply tutorials/secure-nginx-with-let-s-encrypt-on-centos-7/01.en.md]: The result with `--rsa-key-size 4096` on ssllabs.com is:
Key | RSA 4096 bits (e 65537)

File: tutorials/secure-nginx-with-let-s-encrypt-on-centos-7/01.en.md (hunk 1 above)
Comment (1817666) [reply tutorials/secure-nginx-with-let-s-encrypt-on-centos-7/01.en.md]: Done. :)

**Comment by turntin-1817666** [comment]
Note to @n-se 

This entire series of setting up Let's Encrypt has been partly copied from digital ocean articles. 

https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-centos-7

Here step 4 is word to word similar to digital ocean step 4 and step 5 is similar to their step 6.
https://www.diffchecker.com/4ntraIJa

**Even the crontab entry and the text below is similar in both articles.** 

> `15 3 * * * /usr/bin/certbot renew --quiet`
> 
> The 15 3 * * * part of this line means "run the following command at 3:15 am, every day". You may choose any time. 

As said in the #179, even that article seems to be plagiarized from DO. 

_If you guys want to get google search traffic, attempts like this wont get you anywhere._

**Pull request merged**